pom.xml

```diff
@@ -63,7 +63,7 @@
 		<dependency>
 			<groupId>life.qbic</groupId>
 			<artifactId>omero-client-lib</artifactId>
-			<version>1.1.0</version>
+			<version>1.2.0-SNAPSHOT</version>
```

Comment:
do we still want to increase the version?

Reply:
Yes we should increase it to `1.2.0` when we release. `1.1.0` already was released and is the last release there is.